cli/internal/fs/turbo_json.go

```diff
@@ -207,19 +207,19 @@ func (pc Pipeline) HasTask(task string) bool {
 
 // UnmarshalJSON deserializes JSON into a TaskDefinition
 func (c *TaskDefinition) UnmarshalJSON(data []byte) error {
-	rawPipeline := &pipelineJSON{}
-	if err := json.Unmarshal(data, &rawPipeline); err != nil {
+	rawTaskDefinition := &rawTaskDefinition{}
+	if err := json.Unmarshal(data, &rawTaskDefinition); err != nil {
```

Comment:
Nit, but I don't think we need this as a pointer unless I'm missing something. Also, maybe change the variable name to not match the type?
```suggestion
	var rawTaskDefinition rawTaskDefinition
	if err := json.Unmarshal(data, &rawTaskDefinition); err != nil {
```